src/v2/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -78,3 +92,46 @@ const DefaultSuggestion: React.FC<SuggestionItemProps> = ({
     </>
   )
 }
+
+const QuickNavigation: React.FC<{
+  href: string
+  showArtworksButton: boolean
+  showAuctionResultsButton: boolean
+}> = ({ href, showArtworksButton, showAuctionResultsButton }) => {
+  if (!showArtworksButton && !showAuctionResultsButton) return null
+
+  return (
+    <Flex flexWrap="wrap">
+      {!!showArtworksButton && (
+        <QuickNavigationItem to={`${href}/works-for-sale`}>
+          <ArtworkIcon mr={0.5} />
+          Artworks
+        </QuickNavigationItem>
+      )}
+      {!!showAuctionResultsButton && (
+        <QuickNavigationItem to={`${href}/auction-results`}>
+          <AuctionIcon mr={0.5} />
+          Auction Results
+        </QuickNavigationItem>
+      )}
+    </Flex>
+  )
+}
+
+const QuickNavigationItem: React.FC<{ to: string }> = ({ children, to }) => {
+  const onClick = event => {
+    // Stopping the event from propagating to prevent SearchBar from navigation to the main suggestion item url.
+    event.stopPropagation()
+    event.preventDefault()
+
+    // FIXME: Using `window.location.assign(to)` instead of `router.push(to)` to prevent a bug where the search bar won't hide anymore.
+    window.location.assign(to)
+  }
+  return (
+    <ClickableFlex onClick={onClick} mt={1} mr={1}>
```

Comment:
If you want to use `Clickable` get it from palette. This is apparently just a flex with a pointer for a cursor: which should never be used (I'd consider this a bug in the conversation app). Clickable is an actual `<button>` and supports all `BoxProps`

Reply:
To be clear: the reason why we'd use an actual button is [so that it is accessible](https://tech.boxed.com/accessible-ui-stop-making-clickable-divs-264115938716)